services/payments/src/simcore_service_payments/services/payments.py

```diff
@@ -144,40 +144,47 @@ async def acknowledge_one_time_payment(
 
 
 async def on_payment_completed(
-    transaction: PaymentsTransactionsDB, rut_api: ResourceUsageTrackerApi
+    transaction: PaymentsTransactionsDB,
+    rut_api: ResourceUsageTrackerApi,
+    *,
+    notify_enabled: bool,
 ):
     assert transaction.completed_at is not None  # nosec
     assert transaction.initiated_at < transaction.completed_at  # nosec
 
-    _logger.debug(
-        "Notify front-end of payment -> sio SOCKET_IO_PAYMENT_COMPLETED_EVENT "
-    )
-
-    with log_context(
-        _logger,
-        logging.INFO,
-        "%s: Top-up %s credits for %s",
-        RUT,
-        f"{transaction.osparc_credits}",
-        f"{transaction.payment_id=}",
-    ):
-        credit_transaction_id = await rut_api.create_credit_transaction(
-            product_name=transaction.product_name,
-            wallet_id=transaction.wallet_id,
-            wallet_name=f"id={transaction.wallet_id}",
-            user_id=transaction.user_id,
-            user_email=transaction.user_email,
-            osparc_credits=transaction.osparc_credits,
-            payment_transaction_id=transaction.payment_id,
-            created_at=transaction.completed_at,
+    if notify_enabled:
+        _logger.debug(
+            "Notify front-end of payment -> sio SOCKET_IO_PAYMENT_COMPLETED_EVENT "
+            "socketio.notify_payment_completed(sio, user_primary_group_id=gid, payment=transaction)"
```

Comment:
uhm is this this supposed to be in the comment? 😕

Reply:
@GitHK yes, normally this should not happen. But allow me **this time**  a gentle reminder for me in the next PRs to replace with a real notification. In any case this is debug :-)